Module not found: 'main/codecs' in '/lerna-monorepo/node_modules/json-url/src/main'

I'm trying to use lerna to maintain multiple packages. One of my packages uses json-url npm module and when I tried to run my webpack bundle it gave me this error
`Module not found: 'main/codecs' in '/lerna-monorepo/node_modules/json-url/src/main'`

the folder structure is
lerna-monorepo
   --node_modules
      json-url
   --packages
       --mypackage
           <component-using-json-url>

I believe this is due to usage of absolute file referencing in json-url/src/main/index.js

Could you please help me out here

--- a/package.json
+++ b/package.json
@@ -3,7 +3,6 @@
 	"version": "2.6.0",
 	"description": "Compress JSON into compact base64 URI-friendly notation",
 	"main": "dist/node/index.js",
-	"module": "src/main/index.js",
 	"browser": "dist/browser/json-url-single.js",
 	"scripts": {
 		"lint": "eslint src/*",

